src/components/common/FaqAccordion/FaqAcccordion.test.tsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable prettier/prettier */
```

Comment:
why did you disable this ?

Reply:
Eslint configuration with local was giving issues. Fixed now